src/desktop/__tests__/e2e/Onboarding.test.js

```diff
@@ -0,0 +1,107 @@
+describe('Trinity desktop end-to-end', () => {
+    test(
+        'Trinity onboarding existing seed',
+        async () => {
+            const page = await global.__getBrowserPage('onboarding', false);
+
+            // Welcome
+            await page.waitForSelector('#to-seed-intro');
+            await page.click('#to-seed-intro');
+
+            // Seed intro
+            await page.waitForSelector('#to-seed-verify');
+            await page.click('#to-seed-verify');
+
+            // Seed input
+            await page.waitForSelector('.input__editable > div');
+            await page.focus('.input__editable > div');
+            await page.keyboard.type(
+                '999999999999999999999999999999999999999999999999999999999999999999999999999999999',
```

Comment:
```suggestion
                '9'.repeat(81),
```

For easier readability